module/os/freebsd/zfs/vdev_file.c

```diff
@@ -255,14 +255,7 @@ vdev_file_io_start(zio_t *zio)
 			return;
 		}
 
-		switch (zio->io_cmd) {
-		case DKIOCFLUSHWRITECACHE:
```

Comment:
We should go one step father and entirely remove the `sys/dkio.h` headers used in the kernel build.  We do need to keep `lib/libspl/include/sys/dkio.h` for libefi.

```
include/os/linux/spl/sys/dkio.h
include/os/freebsd/spl/sys/dkio.h
```

Reply:
Good call, done!